docs/design/features/byreflike-generics.md

```diff
@@ -248,4 +250,13 @@ class B : A
     // not aware they could be calling B.
     override void M<T2>();
 }
+```
+
+**8) Valid**
+```csharp
+class A
+{
+    public bool M<T, U>(T t) where T: allows ref struct
+        => t is U;
```

Comment:
This sample does not have `t is U u`. Is that intentional? Essentially, that the code gen today only supports type testing not storing into to type?